### DISCUSSION TITLE
华为健康导出的GPX与原来的Garmin数据可以一起显示吗？

### DISCUSSION THREAD

**Discussion by efish2002** [yihong0618/running_page#425] [General]
Garmin的数据我已经走通了，显示也都正常。早期跑步的时候用的华为手表，最近把那些数据导出了GXP格式。现在我把GPX数据放到GPX_OUT文件夹下，和佳明的GPX数据放在一起，然后用RUN_TYPE: only_gpx # 执行，结果还是值显示佳明的数据，我把script中的data.db，删除了重新生成，结果还是只显示佳明的数据，请问这种情况下该如何处理？

**Comment by yihong0618** [comment]
gpx 可以分享一个样例么？应该可以传附件的

**Reply by efish2002** [reply]
https://github.com/efish2002/running_page/blob/master/GPX_OUT/20190702%E6%88%B7%E5%A4%96%E8%B7%91%E6%AD%A5.gpx

**Reply by efish2002** [reply]
在我的项目的GPX_OUT文件夹下，标记为“华为健康导出数据”的

**Comment by yihong0618** [comment]
好的，我看下为什么

**Comment by yihong0618** [comment]
我确认下你 follow 这个顺序

1. 删掉 data.db
2. 删掉 GPX_OUT 里的所有数据
3. 用 gramin_sync 导出所有 gpx
4. 删掉 data.db
5. 把所有华为的 gpx 放进 GPX_OUT
6. 跑 gpx_sync.py
7. 查看网页是否正确

**Reply by yihong0618** [reply]
要删掉两次 data.db 注意。

**Reply by efish2002** [reply]
好的 我再试试，我发现gpx文件也有问题，大小为0

**Reply by yihong0618** [reply]
大小为 0 一定是错的。

**Reply by efish2002** [reply]
是数据有问题，现在可以了，是不是我再把RUN_TYPE：改成garmin-cn就可以了

**Comment by yihong0618** [comment]
对。
